src/AudacityHeaders.h

```diff
@@ -41,6 +29,16 @@
 #include <wx/thread.h>
 #include <wx/tooltip.h>
 
+#include "Audacity.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
```

Comment:
I think it is not necessary to include `stdio.h`, `stdlib.h`, `string.h` and `math.h` here at all. 
These headers are already included by wx.h.

Reply:
Probably they are already included somewhere but, if I can say my opinion, it would be better to not make assumptions about the things that wxWidgets or other libraries could do internally, inside their files.